README.md

```diff
@@ -0,0 +1,35 @@
+# PagerDuty Short Circuiter
+This project contains PagerDuty CLI to reduce the time taken from when SRE receives a PD alert to when troubleshooting on the cluster actually begins.
```

Comment:
Description needs to be refined, describe what's your idea of pdcli for someone who is the first time visitor. 


pdcli is an integration of [go-pagerduty](https://github.com/PagerDuty/go-pagerduty) and [ocm-container](https://github.com/openshift/ocm-container) which lets you spawn ocm-container with automatic cluster login and other features based on the pagerDuty alert.  

**_Note that pdcli is not a reinvention of another pagerDuty CLI tool instead it is a wrapper over go-pagerduty and provides you with all go-pagerduty cli features._**

Features: 
- Users can select the alert, which they want to work upon, through CLI, and they will be able to quickly login to the cluster, using ocm-container, without having to copy-paste information from the alert metadata and will be taken straight to the problematic namespace.
-  Users will be provided with alert metadata in the terminal.
- Can query who is oncall.
- pdcli requires zero configuration, just one-time login is required.
- Sets helpful environment variables inside ocm-container like $NAMESPACE, $JOB, $POD, $INSTANCE, $VERSION  based on the alert metadata.